debezium-connector-oracle/pom.xml

```diff
@@ -168,9 +188,9 @@
         <skipITs>true</skipITs>
 
         <!--
-        Specify properties that will be used for setting up the integration tests' Docker container.
-        Note that the `dockerhost.ip` property is computed from the IP addrses of DOCKER_HOST, which will
-        work on all platforms.  We'll set some of these system properties during integration testing.
+        Specify properties used for setting up the Oracle DB container instance/s for integration tests.
+        Note that the `docker.host.address` property is computed from the IP address of DOCKER_HOST, which will
```

Comment:
@jpechane @Naros is this comment change accurate? I was not able to find any usage of the `dockerhost.ip` parameter.

Reply:
Ya, it seems fine to me, likely just an oversight or maybe a situation where a variable name was adjusted but we failed to keep the comment aligned.